app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -8,7 +8,7 @@ def github
       # See if they are a member of the organization that we have access for
       # If they are, automatically create an account
       client = Octokit::Client.new(access_token: github_token)
-      org_ids = client.organizations.map { |org| org.id.to_s }
+      org_ids = client.organizations.map { |org| org.id }
       if org_ids.include?(github_org_id)
```

Comment:
I would also make sure that `github_org_id` is a Integer and call `github_org_id.to_i`